### PR TITLE
Extended lint applied

### PR DESCRIPTION
**Pull request by doktordirk**

### CODE REVIEW

**Comment by doktordirk** [comment]
@RWOverdijk

File: src/notification.js [hunk 1]
```diff
@@ -219,7 +224,7 @@ export class Notification {
   /**
    * Force remove humane log
    *
-   * @return {Promise}
+   * @return {Promise}         Resolves when closed
```
Comment (RWOverdijk) [inline src/notification.js]: Weird indentation

File: src/notification.js [hunk 2]
```diff
@@ -62,18 +64,15 @@ export class Notification {
...
-
+  info(message, options = {}, defaults = this.__config.defaults) {} // eslint-disable-line  no-empty-function
 
   /**
    * Creates a Notification instance
    *
-   * @param  {[Config]} config
-   * @param  {[Humane]} humane
-   * @param  {[I18N]}   i18N
+   * @param {Config} config The condig instance
```
Comment (RWOverdijk) [inline src/notification.js]: I know you hate doing this, but you made some typos

File: src/notification.js [hunk 3]
```diff
@@ -62,18 +64,15 @@ export class Notification {
...
+   * @param {Humane} humane The humae instance
```
Comment (RWOverdijk) [inline src/notification.js]: Typo

**Review by RWOverdijk — CHANGES_REQUESTED** [review]

File: src/notification.js (hunk 2 above)
Comment (doktordirk) [reply src/notification.js]: that should have been removed anyways seems i missed some

**Review by RWOverdijk — APPROVED** [review]

**Pull request merged**